e2e-tests/cypress/support/insurance/complete-declarations.js

```diff
@@ -13,5 +13,5 @@ export default () => {
   cy.completeAndSubmitDeclarationAntiBriberyCodeOfConduct();
   cy.completeAndSubmitDeclarationAntiBriberyExportingWithCodeOfConduct();
   cy.completeAndSubmitDeclarationConfirmationAndAcknowledgements();
-  cy.completeAndSubmitDeclarationHowYourDataWillBeUsed();
+  // cy.completeAndSubmitDeclarationHowYourDataWillBeUsed();
```

Comment:
reason it is commented out and not removed?

Reply:
Whoops, forgot to undo this - this was just to help with local dev